plugins/org.jkiss.dbeaver.ext.dameng.ui/plugin.xml

```diff
@@ -25,5 +25,17 @@
         <configurator class="org.jkiss.dbeaver.ext.dameng.ui.config.DamengSequenceConfigurator">
             <objectType name="org.jkiss.dbeaver.ext.generic.model.GenericSequence"/>
         </configurator>
+
+        <editor id="source.ddl"
+                class="org.jkiss.dbeaver.ui.editors.sql.SQLSourceViewer"
+                label="%editor.source.ddl.name"
```

Comment:
editor.source.ddl.name and editor.source.ddl.description should have values in the plugins/org.jkiss.dbeaver.ext.dameng.ui/OSGI-INF/l10n/bundle.properties

And yes, you need to create OSGI-INF/l10n/bundle.properties first. Or change %editor.source.ddl.name to the value here.